src/cljs/main/broadfcui/main.cljs

```diff
@@ -175,9 +175,32 @@
    (fn []
      {:user-status #{}})
    :component-will-mount
-   (fn [{:keys [this]}]
+   (fn [{:keys [this state]}]
      (init-nav-paths)
-     (this :handle-hash-change))
+     (this :handle-hash-change)
+     (set! js/window.forceSignedIn
```

Comment:
`js/window.forceSignedIn` -> `(.-forceSignedIn js/window)` to be a bit more idiomatic.